index.js

```diff
@@ -36,16 +37,50 @@ module.exports = (src, dest, opts) => {
 		return Promise.reject(new CpyError('`files` and `destination` required'));
 	}
 
-	return globby(src, opts)
+	const progressEmitter = new EventEmitter();
+	const copyStatus = {};
```

Comment:
Perhaps we could initialise this to `{written: 0, percent: 0}` so that we don't need to do `const fileStatus = copyStatus[event.src] || {written: 0, percent: 0};`?